src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. Instead, set styles only for elements that require them.